packages/web-service/src/pages/habitat/a24/work-end/__tests__/habitat-work-end.spec.js

```diff
@@ -84,72 +84,92 @@ describe('The habitat work end page', () => {
 
   describe('the validator function', () => {
     it('if the user does not input a day - it raises an error', async () => {
-      try {
-        const payload = {
-          'habitat-work-end-day': '',
-          'habitat-work-end-month': '10',
-          'habitat-work-end-year': (new Date().getFullYear())
-        }
-        jest.doMock('../../../../../session-cache/cache-decorator.js', () => {
-          return {
-            cacheDirect: () => {
-              return {
-                getData: () => `11-11-${new Date().getFullYear()}`
-              }
+      const payload = {
+        'habitat-work-end-day': '',
+        'habitat-work-end-month': '10',
+        'habitat-work-end-year': (new Date().getFullYear())
+      }
+      jest.doMock('../../../../../session-cache/cache-decorator.js', () => {
+        return {
+          cacheDirect: () => {
+            return {
+              getData: () => `11-11-${new Date().getFullYear()}`
```

Comment:
I am lost why this needs to be set like this without a globally set mock time ? 

Set the default time in a before each / all and then there is no need for any of the new date logic ?